internal/services/cdn/cdn_frontdoor_firewall_policy_resource.go

```diff
@@ -0,0 +1,1095 @@
+package cdn
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/location"
+
+	"github.com/Azure/azure-sdk-for-go/services/frontdoor/mgmt/2020-11-01/frontdoor"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
```

Comment:
Can we sort imports here?
```suggestion
	"github.com/hashicorp/go-azure-helpers/resourcemanager/location"
	"github.com/Azure/azure-sdk-for-go/services/frontdoor/mgmt/2020-11-01/frontdoor"
	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
```

Reply:
Fixed.